src/flow/store/event/eventActionLazy.js

```diff
@@ -403,3 +404,55 @@ export function removeConferencingRoom(url) {
     setTimeout(() => dispatch(removeConferencingRoomAction('removed')), 1000)
   }
 }
+
+export function verifyCalendarAction(payload) {
```

Comment:
convention in this project is to not use the suffix `Action`.
Better naming `setCalendarVerificationStatus`

Reply:
I'm just follow the application pattern... I saw others actions with this naming...
![image](https://user-images.githubusercontent.com/294406/52783642-b0dc2f80-3039-11e9-85df-7608846cd5bd.png)

But, I changed now to setCalendarVerificationStatus as you asked.

Reply:
thank you